### PR TITLE
[WIP] Split REST API integration files up

### PR DESCRIPTION
**Pull request by Nana-EC**
**Detailed description**:
REST API tests have been hard to decipher, modify and are sometimes not consistent.
These changes provide some scaffolding and changes to improve the integration tests on the REST side

- Breaks out integration.test.js into 2 files. One to handle url query tests and other to handle dbquery tests
- Updated files names to be camelCase

**Which issue(s) this PR fixes**:
Partially addresses #412 with inspiration from #529

**Special notes for your reviewer**:
This PR takes a chunk of #594 changes to make reviewing simpler

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: hedera-mirror-rest/__tests__/integration.test.js [hunk 1]
```diff
@@ -27,10 +27,8 @@
  * schema (using sql files in the ../src/resources/db/migration directory).
  *
  * Test data is created by:
- * 1) reading account id, balance, expiration and crypto transfer information from integration_test_data.json
- * 2) storing those accounts in integration DB
- * 3) creating 3 balances records per account at timestamp 1000, 2000, 3000 in the integration DB
- * 4) apply transfers (from integration_test_data.json) to the integration DB
+ * 1) reading account id, balance, expiration and crypto transfer information from *.spec.json
+ * 2) storing those accounts in integration DB      * 2) apply account creations, balance sets and transfers to the integration DB
```
Comment (apeksharma) [inline hedera-mirror-rest/__tests__/integration.test.js]: fix comment in followup.

File: hedera-mirror-rest/__tests__/integrationDbQuery.test.js [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+
+//
+// TEST DATA
+// shard 0, realm 15, accounts 1-10
+// 3 balances per account
+// several transactions
+//
+const shard = 0;
+const realm = 15;
+const addAccount = async function(accountId, exp_tm_nanosecs = null) {
```
Comment (apeksharma) [inline hedera-mirror-rest/__tests__/integrationDbQuery.test.js]: in followup, let's share the below common functions.

Comment (Nana-EC) [reply hedera-mirror-rest/__tests__/integrationDbQuery.test.js]: These methods are about to be removed. They are only repeated here in this interim PR per your request.

File: hedera-mirror-rest/__tests__/integration.test.js (hunk 1 above)
Comment (Nana-EC) [reply hedera-mirror-rest/__tests__/integration.test.js]: Fixed.

File: hedera-mirror-rest/__tests__/integration.test.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
  * ‌
  * Hedera Mirror Node
  * ​
- * Copyright (C) 2019 Hedera Hashgraph, LLC
+ * Copyright (C) 2020 Hedera Hashgraph, LLC
```
Comment (steven-sheehy) [inline hedera-mirror-rest/__tests__/integration.test.js]: Files should contain a range of years they are copyrighted instead of being latest.

Comment (Nana-EC) [reply hedera-mirror-rest/__tests__/integration.test.js]: Fixed

**Comment by Nana-EC** [comment]
Blocked on flyway db issues in circle when there are 2 integration files. Going to hold off on splitting the files. Will put out a new PR with setup logic

**Comment by Nana-EC** [comment]
Reopening just to test out circle ci and flyway logic changes. Should I get it working I'll create a separate PR with the current updated master code

**Comment by Nana-EC** [comment]
Not a priority right now. Can come back to trying to split the files should it be a desire.

**Pull request closed without merging**